### DISCUSSION TITLE
Lenght of text area in custom fields

### DISCUSSION THREAD

**Discussion by jongerius** [creecros/MetaMagik#92] [Q&A]
Goodday,
I,m working with kanboard for a few weeks now and i like it very much. Unfortanaly i'm not a programmer, but github and google helps me a lot. I have managed te use Metamagik and add custom fields, very lovely but: When i create a text area, the maximum length of text is 255. Can i change that somewhere? I want for instance 1000 

When i type more then i get this error message:

Internal Error: SQL Error[22001]: SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'value' at row 1

Best regards,

**Comment by creecros** [comment]
You would have to change the  type of data of "task_has_metadata" column in your database.

This is a core column, not defined by the plugin.

What to change it to, would also depend on what database you use.

**Reply by creecros** [reply]
For instance, the mysql schema is defined as
https://github.com/kanboard/kanboard/blob/224a15692c11db2ac06183d4975dfffa28c68d12/app/Schema/Mysql.php#L614

Yet, sqlite, would not have the 255 char limit.

**Reply by jongerius** [reply]
Thanks for your fast respond! I'll try to find out.
